src/reducers/requests/jobs.js

```diff
@@ -1,59 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-// @flow
-
-import {combineReducers} from 'redux';
-import {JobTypes} from 'action_types';
-
-import {handleRequest, initialRequestState} from './helpers';
-
-import type {GenericAction} from 'types/actions';
-import type {JobsRequestsStatuses, RequestStatusType} from 'types/requests';
-
-function createJob(state: RequestStatusType = initialRequestState(), action: GenericAction): RequestStatusType {
-    return handleRequest(
-        JobTypes.CREATE_JOB_REQUEST,
```

Comment:
sure can

Reply:
Actually @sudheerDev `src/action_types/jobs.js` is imported in `src/actions/jobs.js` and the variables are used in the requests so I dont think we can delete this file.

Reply:
Sorry i didn't realise that but those actions can be removed from `src/actions/jobs.js` because we don't need them either. Then you will be able to remove the `action_types` file

Reply:
For reference you can check https://github.com/mattermost/mattermost-redux/pull/940